### PR TITLE
[Core] Add missing channel configuration exception

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Component/Core/spec/Exception/ChannelNotDefinedExceptionSpec.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+use Sylius\Component\Core\Exception\ChannelNotDefinedException;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+final class ChannelNotDefinedExceptionSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith(ChannelNotDefinedException::class);
```
Comment (pamil) [inline src/Sylius/Component/Core/spec/Exception/ChannelNotDefinedExceptionSpec.php]: ?

**Comment by michalmarcinkowski** [comment]
I think we should also validate configuration in admin panel, so that you must specify the amount for channels in which the shipping method is enabled.

**Comment by GSadee** [comment]
@michalmarcinkowski I've checked that and we already have this validation in admin panel, so this is just an additional security.

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu!

**Pull request merged**